db/migrate/20170409170753_add_last_webfingered_at_to_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastWebfingeredAtToAccounts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :accounts, :last_webfingered_at, :datetime, null: true, default: nil
```

Comment:
If `null: true` and `default: nil`, then we can actually remove both options. All columns are by definition without a null constraint and by default nil.